### DISCUSSION TITLE
NestJS schedulers are not working in production

### DISCUSSION THREAD

**Discussion by RoyLeibo** [vercel/community#1382] [Help]
### Summary

I have a BE service in NestJS that is deployed in Vercel.

I need several schedulers, so I have used @nestjs/schedule lib, which is super easy to use.

Locally, everything works perfectly. For some reason, the only thing that is not working in my production environment is those schedulers. Everything else is working - endpoints, data base access..

Does anyone has an idea why? is it something with my deployment? maybe Vercel has some issue with that? maybe this schedule library requires something the Vercel doesn't have?

I am clueless..

**Comment by dadwic** [comment]
Hey, I had the same issue with the NestJS schedule module. use [Vercel Cron Jobs](https://vercel.com/guides/how-to-setup-cron-jobs-on-vercel)
